src/hotspot/share/runtime/threadWXSetters.inline.hpp

```diff
@@ -30,20 +30,16 @@
 
 #if defined(__APPLE__) && defined(AARCH64)
 
-#include "runtime/thread.inline.hpp"
+#include "runtime/thread.inline.hpp" // dependencies require this include
```

Comment:
I can't see how this include is needed now.

Reply:
I tried to replace it with os.hpp (and os.inline.hpp) but it caused a linker error.

Reply:
I was wrong, I removed it.